news/WildFly-s2i-19-released.adoc

```diff
@@ -0,0 +1,148 @@
+= WildFly 19 S2I image has been released on quay.io
+jfdenise
+2020-03-20
+:awestruct-tags: [wildfly, galleon]
+:awestruct-layout: blog
+:source-highlighter: coderay
+:encoding: utf-8
+:lang: en
+
+==  WildFly 19 S2I docker images
+
+The WildFly s2i builder and runtime Docker images for WildFly 19 have been released on link:https://quay.io/organization/wildfly[quay.io/wildfly].
+
+Changes since the link:https://wildfly.org/news/2019/10/07/WildFly-s2i-18-released/[last release]:
+
+* Optimized startup. The server starts much faster, as the CLI script applied to the server configuration is now executed during server boot.
+* Support for clustering. JGroups is now part of the default server configuration. 
+* The WildFly 19 Galleon decorator layer _web-clustering_ (support for Infinispan-based web session clustering) is fully usable in the openshift context.
```

Comment:
I'm nitpicking so feel free to ignore me but I think everything that ends up in code or shell script should be properly formatted as `web-clustering`, not _web-clustering_. This is consistent with code block and ensure that emphasis display is used only when some text needs to be emphasized